CHANGELOG.md

```diff
@@ -2,31 +2,80 @@
-
-The header file is now named `rustls.h` instead of `crustls.h`, and the
-generated `.a` file is named `librustls_ffi.a` instead of `libcrustls.a`.
+The package name has changed to "rustls-ffi" (from "crustls").
+The header file (as installed by `make DESTDIR=/path/ install`)
+is now `rustls.h` and the library is `librustls.a`. The old library and header
+names are symlinked as part of the install process, to simplify upgrading to the
+new version.
+
+If you are importing this as a library from other Rust code, you should import `rustls_ffi`.
+
+## New
+ - rustls_{client,server}_config_builder_new_custom: start building a config,
+   with ciphersuites and TLS versions set at initial construction.
+ - rustls_default_ciphersuites_get_entry() and
+   rustls_default_ciphersuites_len(): get default ciphersuites as opposed to
+   all ciphersuites (these happen to be the same today but might not always be).
+
+## Changed
+
+- `rustls-ffi` now imports `rustls` version 0.20, up from rustls 0.19. [View
+  the changelog](https://github.com/rustls/rustls#release-history).
+- Configuring ciphersuites and TLS versions. Previously these
+  could be set using setter methods on the builder object. Now they have
+  to be set at the beginning of the config builder process, by calling
+  rustls_client_config_builder_new_custom().
+- Reading of plaintext from a rustls_connection. When the
+  internal plaintext buffer is empty, rustls_connection_read will return
+  RUSTLS_RESULT_PLAINTEXT_EMPTY. That means no more plaintext can be read until
+  additional TLS bytes are ingested via rustls_connection_read_tls, and
+  rustls_connection_process_new_packets is called. Previously this condition was
+  indicated by returning RUSTLS_RESULT_OK with out_n set to 0.
+- Handling of unclean close and the close_notify TLS alert. Mirroring upstream changes,
+  a rustls_connection now tracks TCP closed state like so: rustls_connection_read_tls
+  considers a 0-length read from its callback to mean "TCP stream was closed by peer."
+  If that happens before the peer sent close_notify, rustls_connection_read will return
+  RUSTLS_RESULT_UNEXPECTED_EOF once the available plaintext bytes are exhausted. This is
+  useful to protect against truncation attacks. Note:
+  some TLS implementations don't send close_notify. If you are already getting length
+  information from your protocol (e.g. Content-Length in HTTP) you may choose to
+  ignore UNEXPECTED_EOF so long as the number of plaintext bytes was as
+  expected.
+- `rustls_version` returns a `rustls_str` that points to a static string in
+  memory, and the function no longer accepts a character buffer or length.
+- Some errors starting with RESULTS_RESULT_CERT_ have been removed, and
+  some renamed.
+- rustls_client_config_builder_set_protocols and
+  rustls_server_config_builder_set_protocols are now
+  rustls_{client,server}_config_builder_set_alpn_protocols.
```

Comment:
Same note here about explicitly writing out `rustls_server_config_builder_set_alpn_protocols` and `rustls_client_config_builder_set_alpn_protocols`